README.md

```diff
@@ -97,11 +97,11 @@ at large cloud providers.
 Operating System | MariaDB | PostgreSQL
 ---------------- | --------| ---------------
 CentOS Linux 7   | 5.5     | *not supported*
-Debian 11        | 10.5    | 13.7
+Debian 11        | 10.5 ?? | 13.7 ??
 Docker           | n/a     | n/a
-FreeBSD 13.0     | 5.7 (*) | 13.6
-Ubuntu 22.04     | 10.5    | 14.2
-Rocky Linux 8.6  | 10.3    | 10.19
+FreeBSD 13.1     | 5.7 (*) | 13.9
+Rocky Linux 9.1  | 10.5 ?? | 13.??
+Ubuntu 22.04     | 10.6    | 14.2 ??
```

Comment:
The version of PG for Ubuntu seems to be 14.5 (https://packages.ubuntu.com/jammy/postgresql-14)